src/java.base/share/classes/java/time/LocalDateTime.java

```diff
@@ -1683,7 +1683,7 @@ public Temporal adjustInto(Temporal temporal) {
     @Override
     public long until(Temporal endExclusive, TemporalUnit unit) {
         LocalDateTime end = LocalDateTime.from(endExclusive);
-        if (unit instanceof ChronoUnit) {
+        if (unit instanceof ChronoUnit u) {
```

Comment:
`chronoUnit` is perhaps a better variable name than `u`

Reply:
Thanks for your comments, @forax, and apologizes for the delay in getting back to you. I was waiting for the boot JDK version to be updated to 16. Certain files changed in the PR are shared between the build tool and the JDK runtime, and were causing build issues.
I've addressed the changes you suggested, and you can find them in commit 647bd6b